src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1284,52 +1284,58 @@ namespace DataSurfaces {
         }
     };
 
+    struct GroundSurfacesData
+    {
+        // Members
+        std::string Name;        // name of a ground surface
+        Real64 ViewFactor = 0.0; // view factor to a ground surface
+        int TempSchPtr = 0;      // pointer to a ground surface temperature schedule object
+        int ReflSchPtr = 0;      // pointer to a ground Surface reflectance schedule object
+    };
+
+    struct GroundSurfacesProperty
+    {
+        // Members
+        std::string Name;                     // name of multiple ground surfaces object
+        int NumGndSurfs = 0;                  // number of groundSurfaces
+        Array1D<GroundSurfacesData> GndSurfs; // ground surfaces data
```

Comment:
If you are creating structures from scratch it would be better to just use raw `std::vector` or at least `EPVector`.  We're always on a mission to reduce our dependence on `Array1D` for simple list use cases.